src/main/java/org/radarcns/management/web/rest/MetaTokenResource.java

```diff
@@ -38,16 +42,16 @@
     @GetMapping("/meta-token/{tokenName:" + Constants.TOKEN_NAME_REGEX + "}")
     @Timed
     public ResponseEntity<TokenDTO> getTokenByTokenName(@PathVariable("tokenName") String tokenName)
-            throws MalformedURLException {
+            throws MalformedURLException, CustomNotFoundException {
         log.info("Requesting token with tokenName {}", tokenName);
-        TokenDTO retrievedToken = metaTokenService.fetchToken(tokenName);
-
         try {
-            TimeUnit.SECONDS.sleep(1);
+            // sleep for 5 seconds
+            TimeUnit.SECONDS.sleep(5);
+            TokenDTO retrievedToken = metaTokenService.fetchToken(tokenName);
+            return ResponseEntity.ok().body(retrievedToken);
         } catch (InterruptedException e) {
             Thread.currentThread().interrupt();
+            return ResponseEntity.unprocessableEntity().build();
```

Comment:
I’d expect HTTP status code 500 here, as unexpected server state.

Reply:
Sure. Will change the status code